src/main/resources/application.yaml

```
@@ -50,4 +50,9 @@ management:
 
 server:
   port: ${PORT:8080}
-  address: 0.0.0.0
\ No newline at end of file
+  address: 0.0.0.0
+
+naver:
+  api:
+    client-id: w77BI41SmIuc2Nfm3L4d
+    client-secret: nDx6IuyH3i
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)